packages/matchbox/src/components/InlineCode/InlineCode.tsx

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { Box } from '../Box';
 
-function InlineCode({ children }) {
+function InlineCode({ children }): JSX.Element {
```

Comment:
There's a warning here about this not being typed 
![Screen Shot 2021-08-20 at 8 42 11 AM](https://user-images.githubusercontent.com/3903325/130234727-62ad0b59-245f-4725-ac96-3ef122c1779e.png)